tool/generate_font.dart

```diff
@@ -6,6 +6,7 @@ import 'package:version/version.dart';
 
 const Map<String, String> nameAdjustments = {
   "500px": "fiveHundredPx",
+  "360-degrees": "threeSixtyDegrees",
```

Comment:
See comment in `generate_example.dart`
```suggestion
  "360-degrees": "threeHundredSixtyDegrees",
```